utils/zerovec/src/ule/tuplevar.rs

```diff
@@ -30,17 +31,17 @@ macro_rules! tuple_varule {
         #[doc = concat!("VarULE type for tuples with ", $len, " elements. See module docs for more information")]
         #[repr(transparent)]
         #[allow(clippy::exhaustive_structs)] // stable
-        pub struct $name<$($T: ?Sized),+> {
+        pub struct $name<$($T: ?Sized,)+ Format: VarZeroVecFormat = Index16> {
```

Comment:
Suggestion: make the format explicit?

I'm OK with the default for higher-level types, but for a lower-level type, let's just make everyone pick the right value? And this is a highly unsafe type, so being explicit seems safer.

Reply:
Why is this a highly unsafe type?

Reply:
You mean impl-wise? I don't think the format shows up too often in the unsafe code here.

I'd rather not have most users care about the format. I don't think this is too low-level, it's a tuple.

Reply:
ok